### PR TITLE
update version for fixing appending slash issue

### PR DESCRIPTION
**Pull request by XIWEI-house**

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -3,7 +3,9 @@ All notable changes to the Chartbeat iOS SDK  will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
+## [1.4.3] - 2021-02-18
+### Added
+- Addressed issue with appending slash on viewId in Iphone SDK.
```
Comment (RonnieBrooklyn) [inline CHANGELOG.MD]: nit: this should say prepending? right? Also, this change log is only for iphone sdk so no need to mention iphone.

**Review by RonnieBrooklyn — COMMENTED** [review]
one nit. feel free to merge when fixed.

File: CHANGELOG.MD (hunk 1 above)
Comment (XIWEI-house) [reply CHANGELOG.MD]: Yup prepending is better wording here, updating

**Comment by XIWEI-house** [comment]
For `versions/A` directory, I copied it from Iphone-SDK to chartbeat-ios-sdk repo.

**Pull request merged**